aiounifi/devices.py

```diff
@@ -49,6 +49,7 @@ def __init__(
         """Initialize device."""
         super().__init__(raw, request)
         self.ports = Ports(raw.get("port_table", []))
+        self.outlets = Outlets(raw.get("outlet_table", []))
```

Comment:
```suggestion
        self.outlets = Outlets(raw.get("outlet_table", []))
```
Make sure to do the same after line 61 to make it possible to update states with the strip as well